application.properties

```
@@ -121,8 +121,8 @@ server.http2.enabled=true
 
 server.ssl.enabled=true
 
-server.ssl.ciphers=${TOMCAT_SSL_CIPHERS:-HIGH:!ADH:!EXP:!SSLv2:!SSLv3:!MEDIUM:!LOW:!NULL:!aNULL}
-server.ssl.enabled-protocols=${TOMCAT_SSL_ENABLED_PROTOCOLS:-TLSv1.3,+TLSv1.2}
+server.ssl.ciphers=${TOMCAT_SSL_CIPHERS:-HIGH:!aNULL:!eNULL:!EXPORT:!DES:!RC4:!MD5:!kRSA}
+server.ssl.enabled-protocols=${TOMCAT_SSL_ENABLED_PROTOCOLS:-TLSv1.3,TLSv1.2}
```

The doesn't match the value above, and it breaks the server when I try it. I also think we want TLS 1.3 enabled.

```suggestion
server.ssl.enabled-protocols=${TOMCAT_SSL_ENABLED_PROTOCOLS:TLSv1.3,TLSv1.2}
```

---

I removed the + on purpose, as it looked like a long-standing typo.

the - in that context is supposed to tell it to use what follows as a default if the var isn't defined. How did it break yours?

---

Ah, OK. I misunderstood the syntax. I tried setting my `application.properties` to use `server.ssl.enabled-protocols=-TLSv1.3,TLSv1.2` which broke things, not realizing that the `-` wasn't part of the value that got passed through.